src/olympia/reviewers/templates/reviewers/abuse_reports.html

```diff
@@ -1,7 +1,7 @@
 {% extends "reviewers/base.html" %}
 
 {% block title %}
-  {{ reviewer_page_title(_('{addon} :: Abuse Reports')|format_html(addon=addon.name)) }}
+  Abuse Reports – {{ addon.name }} – Add-ons for Firefox
```

Comment:
you'll need to make this string translatable they should be wrapped in `{{ _('....') }}` with the format filter, e.g. `{% block title %}{{ _('  Abuse Reports – {addon} – Add-ons for Firefox')|format_html(addon=addon.name) }}{% endblock %}`